package.json

```diff
@@ -63,21 +62,22 @@
     "ember-resolver": "^8.0.3",
     "ember-source": "~3.28.4",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.10.0",
+    "ember-template-lint": "^6.0.0",
     "ember-try": "^1.4.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.7",
+    "eslint": "^9.14.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^12.3.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.0.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-qunit": "^7.3.4",
+    "globals": "^15.12.0",
     "loader.js": "^4.7.0",
-    "mocha": "^9.1.3",
-    "prettier": "^2.4.1",
-    "qunit": "^2.17.2",
+    "mocha": "^9.2.2",
+    "prettier": "^3.3.3",
```

Comment:
@cloke I'll try to create separate PR today to bump `prettier` and `eslint-*-prettier` packages to keep this one focused solely on ESLint